.github/workflows/build-report.yml

```diff
@@ -1,9 +1,9 @@
-name: Generate Weekly Test Report
+name: Generate Test Report Table
 
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 16 * * *"  # 4pm UTC = 8am PST / 9am PDT, 7 hours after testapps run
+    - cron: "0 15 * * *"  # 3pm UTC = 7am PST / 8am PDT, 6 hours after testapps run
```

Comment:
Any reason for the change?

Reply:
Since it takes ~1 hour to run, I wanted to ensure it was finished by 10am Pacific (in case it happened to take longer). 6 hours should be enough for the testapps.